Sources/Parsers/PlistParser.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  PlistParser.swift
+//  SwiftGenKit
+//
+//  Created by Toshihiro Suzuki on 4/12/17.
+//  Copyright © 2017 SwiftGen. All rights reserved.
+//
+
+import Foundation
+import PathKit
+
+public final class PlistParser {
+
+  var keys: [String] = []
```

Comment:
Might be more interesting to also load & store type information for each key. Another thing you might want to do: nice names for keys (like the Xcode plist editor).

Also: try to support multiple files, like the other parsers.